components/TableColumns/TableColumns.tsx

```diff
@@ -90,6 +105,287 @@ export const columns: ColumnDef<Header>[] = [
      */
     cell: ({ row }) => <div>{row.getValue('text3')}</div>,
   },
+  {
+    accessorKey: 'text4',
+
+    /**
+     * Value of row.
+     * @param {object} column - The column data.
+     * @param {object} column.column - The column definition
+     * @returns {JSX.Element} - The cell component.
+     */
+    header: ({ column }): JSX.Element => {
+      return (
+        <Button
+          variant="ghost"
+          onClick={() => column.toggleSorting(column.getIsSorted() === 'asc')}
+        >
+          Surviving?
+          <ChevronsUpDown className="ml-2 h-4 w-4" />
+        </Button>
+      );
+    },
+
+    /**
+     * Value of row.
+     * @param {object} row - The row data.
+     * @param {object} row.row - The row definition
+     * @returns {JSX.Element} - The cell component.
+     */
+    cell: ({ row }) => <div>{row.getValue('text2')}</div>,
+  },
+  {
+    id: 'actions',
+
+    /**
+     * Admin action dropdown.
+     * @returns {JSX.Element} - The cell component.
+     */
+    cell: (): JSX.Element => {
+      return (
+        <DropdownMenu>
+          <DropdownMenuTrigger asChild>
+            <Button variant="ghost" className="h-8 w-8 p-0">
+              <span className="sr-only">Open menu</span>
+              <MoreHorizontal className="h-4 w-4" />
+            </Button>
+          </DropdownMenuTrigger>
+          <DropdownMenuContent align="end">
+            <DropdownMenuItem>View</DropdownMenuItem>
+            <DropdownMenuItem>Edit</DropdownMenuItem>
+            <DropdownMenuItem>Delete</DropdownMenuItem>
+          </DropdownMenuContent>
+        </DropdownMenu>
+      );
+    },
+  },
+];
+
+export const leagueColumns: ColumnDef<LeagueHeader>[] = [
+  {
+    accessorKey: 'text',
+    header: 'League Name',
+    /**
+     * Value of row.
+     * @param {object} row - The row data.
+     * @param {object} row.row - The row definition
+     * @returns {JSX.Element} - The cell component.
+     */
+    cell: ({ row }) => <div>{row.getValue('text')}</div>,
+  },
+  {
+    accessorKey: 'text2',
+
+    /**
+     * Value of row.
+     * @param {object} column - The column data.
+     * @param {object} column.column - The column definition
+     * @returns {JSX.Element} - The cell component.
+     */
+    header: ({ column }): JSX.Element => {
+      return (
+        <Button
+          variant="ghost"
+          onClick={() => column.toggleSorting(column.getIsSorted() === 'asc')}
+        >
+          # of Participants
```

Comment:
nit: How come this isn't just Number of Participants or num of Participants?

Reply:
personal preference. I went with this as it will show up on the table. Keeps things short and easier to read. Maybe Shashi would want a different title